src/main/java/beam/analysis/plots/filterevent/ActivitySimFilterEvent.scala

```diff
@@ -0,0 +1,47 @@
+package beam.analysis.plots.filterevent
+
+import scala.collection.JavaConverters._
+
+import beam.agentsim.events.{ModeChoiceEvent, ReplanningEvent}
+import beam.sim.config.BeamConfig
+import org.matsim.api.core.v01.events.Event
+import org.matsim.api.core.v01.population.{Activity, PlanElement}
+import org.matsim.core.controler.MatsimServices
+
+class ActivitySimFilterEvent(beamConfig: BeamConfig, matsimServices: MatsimServices) extends FilterEvent {
```

Comment:
Please, add some tests to verify functionality

Reply:
added